statsmodels/tsa/statespace/tests/test_sarimax.py

```diff
@@ -397,14 +399,14 @@ def __init__(self):
         true['se_exog'] = []
 
         super(TestFriedmanStateRegression, self).__init__(
-            true, mle_regression=False
+            true, exog=exog, mle_regression=False
         )
 
         self.result = self.model.filter()
 
     def test_regression_parameters(self):
         assert_almost_equal(
-            self.result.filtered_state[-2:, -1],
+            self.result.filtered_state[-2:, -1] / 10.,
```

Comment:
can you add a comment what parameter this refers to?
I had problems before understanding parts of the unit tests